test/controllers/questions_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class QuestionsControllerTest < ActionController::TestCase
+  setup do
+    @question = questions(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:questions)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create question" do
+    assert_difference('Question.count') do
+      post :create, question: { description: @question.description, resolve: @question.resolve, title: @question.title, user_id: @question.user_id }
+    end
+
+    assert_redirected_to question_path(assigns(:question))
+  end
+
+  test "should show question" do
+    get :show, id: @question
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @question
+    assert_response :success
+  end
+
+  test "should update question" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.